src/components/views/settings/tabs/user/SessionManagerTab.tsx

```diff
@@ -173,7 +173,10 @@ const SessionManagerTab: React.FC = () => {
      * delegated auth provider.
      * See https://github.com/matrix-org/matrix-spec-proposals/pull/3824
      */
-    const delegatedAuthAccountUrl = sdkContext.oidcClientStore.accountManagementEndpoint;
+    const delegatedAuthAccountUrl = useAsyncMemo(async () => {
+        await sdkContext.oidcClientStore.readyPromise; // wait for the store to be ready
+        return sdkContext.oidcClientStore.accountManagementEndpoint;
+    }, [sdkContext.oidcClientStore]);
     const disableMultipleSignout = !!delegatedAuthAccountUrl;
```

Comment:
I haven't read through the whole class in detail, but won't setting `delegatedAuthAccountUrl` to `undefined` while we make a bunch of network requests mean that we display the wrong UI for a while?  We need to distinguish between "we don't know yet" and "we're definitely not OIDC".

Reply:
Indeed, no design was given for this case so this is what we're going with for the time being.